### PR TITLE
apps/cms: Simplify handling of encerts; add warning if they are ignored

### PR DESCRIPTION
**Pull request by DDvO**
SSIA

### CODE REVIEW

**Comment by DDvO** [comment]
AppVeyor failure is unrelated.

**Comment by DDvO** [comment]
Rebased to fix merge conflict.
Can we please get this done?

File: apps/cms.c [hunk 1]
```diff
@@ -301,7 +301,7 @@ int cms_main(int argc, char **argv)
     OPTION_CHOICE o;
     OSSL_LIB_CTX *libctx = app_get0_libctx();
 
-    if ((vpm = X509_VERIFY_PARAM_new()) == NULL)
+    if (encerts == NULL || vpm == NULL)
```
Comment (t8m) [inline apps/cms.c:304]: Use goto end; otherwise it theoretically can leak one of those.

**Review by t8m — CHANGES_REQUESTED** [review]
One minor fix needed.

File: apps/cms.c (hunk 1 above)
Comment (DDvO) [reply apps/cms.c:304]: Ah, right; done.
And rebased to fix merge conflicts.

**Comment by DDvO** [comment]
@t8m, is this okay now?

**Review by t8m — APPROVED** [review]

**Comment by DDvO** [comment]
Merged - thanks @t8m

**Pull request closed without merging**